src/Route/FastPsr7RouteInfo.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Penny\Route;
+
+class FastPsr7RouteInfo implements RouteInfoInterface
+{
+    private $name;
+    private $callable;
+    private $params;
+
+    public static function matched($name, callable $callable, $params = [])
```

Comment:
Why this method?

```php
public function __construct($name, callable $callable, $params = []) {
    // ...
}
```

Is not better? In that way you can simplify the RouteInfoInterface and let to the user the RouteInfo construction